io/module.js

```diff
@@ -93,13 +93,17 @@ const register = {
     } = options
 
     const { ext: ioSvcExt } = pParse(ioSvc)
-    const { ext: nspDirExt } = pParse(ioSvc)
+    const { ext: nspDirExt } = pParse(nspDir)
```

Comment:
I prefer to keep this as pParse(ioSvc) because ioSvc could be typed as either a directory or filename. The nspDir is designed to always be a directory

Reply:
`nspDir` value not always directory if set `ioSvc` with ext and not set `nspDIr` .
https://github.com/richardeschloss/nuxt-socket-io/blob/dbb73a48d6c52cdaf43f90c182b0f49cc0d46b24/io/module.js#L88-L89

This code check `nspDir` copy from `ioSvc(with ext)` pattern.
```suggestion
      extList.includes(nspDirExt)
        ? nspDir.substr(0, nspDir.length - nspDirExt.length)
        : nspDir
```

Reply:
I will your repo resubmit fix/224 branch if you agree to this code change.